logic/scorer/scorer.go

```diff
@@ -23,26 +37,23 @@ func points(lead model.Card, hand []model.Card, isCrib bool) int {
 		return 0
 	}
 
-	values := make([]int, 5)
-	ptValues := make([]int, 5)
+	var vals, ptVals [numCardsToScore]int
```

Comment:
:cactus: :book:  can we keep this as `values` and `ptValues` to reduce the diff? :#